CONTRIBUTING.md

```diff
@@ -97,44 +92,52 @@ $ git push origin issue_38
 ```
 
 * Push changes to GitHub.
-* Create pull request from GitHub webpage (your fork's dashboard).
+* Create pull request from GitHub (your fork's dashboard).
 
 ### Branch naming conventions
-We prefer you to use standardised naming convention when creating pull requests.
-Here are a few examples of branch name prefixes you could use when creating a pull request from your fork:
-* ```issue_``` - branch with fix for issue. E.g. ```issue_38```.
-* ```doc_``` - documentation update. E.g. ```doc_add_faq```.
-* ```devel_``` - development of a new feature. E.g. ```devel_udp_client_test```.
-* ```test_``` - when pull request will consist of only new/updates to test cases. E.g. ```test_paralllel_execution```.
+
+We prefer you use a standard naming convention when creating pull requests. Here are a few examples of branch name prefixes you could use when creating a pull request from your fork:
+
+* `issue_` - branch with fix for issue. For example, `issue_38`.
+* `doc_` - documentation update. For example, `doc_add_faq`.
+* `devel_` - development of a new feature. For example, `devel_udp_client_test`.
+* `test_` - when pull request will consist of only new/updates to test cases. For example, `test_paralllel_execution`.
 
 ## Coding style and coding rules
-This chapter attempts to explain the basic styles and patterns that are used in mbed test tools projects. The following rules should be followed for producing new code and updating existing code.
 
-### Code Like a Pythonista: Idiomatic Python
-Please do your best to follow [Idiomatic Python](http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html) interactive tutorial.
+This chapter attempts to explain the basic styles and patterns in mbed test tools projects. Do the following to produce new code and update existing code.
+
+### Code like a Pythonista: idiomatic Python
+
+Please do your best to follow the [Idiomatic Python](http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html) interactive tutorial.
+
+### Coding style: readability matters
```

Comment:
At this point, seems like this section is really necessary, perhaps we could remove it?